kit/azure/meshplatform/documentation.tf

```diff
@@ -0,0 +1,20 @@
+output "documentation_md" {
+  value = <<EOF
+# meshStack Integration
+
+meshStack integration sets up this AAD tenant as a meshPlatform.
+To do this we create AAD tenant-level service principals, allowing meshStack to access data
+and orchestrate Azure platform functionality.
+
+## Replicator Service Principal
+ - Application Client ID: ${module.meshplatform.replicator_service_principal["Application_Client_ID"]}
+ - Enterprise Application Object ID: ${module.meshplatform.replicator_service_principal["Enterprise_Application_Object_ID"]}
+ - Client_Secret:  ${module.meshplatform.replicator_service_principal["Client_Secret"]}
```

Comment:
f: do not put secrets into emitted documentation page!

Reply:
its not adding secrets :) its add a text how to get the secret if u needing it